### PR TITLE
Show the users' federated name in drawer.

### PR DESCRIPTION
**Pull request by camporter**
This lets users know which instance they are on more easily.

Also change the 'Switch to' name/server format to match the federated name format using `@`.

federated_name_in_drawer

### CODE REVIEW

**Comment by MV-GH** [comment]
What happens if the name is way too long? Does it completely clip it out? If so you should probably change the text clipping to do the ellipsis thing.

**Comment by camporter** [comment]
The federated name was previously given one line and clip. Gave it two and ellipsis and it seems like that should be plenty 🙂 

The existing `PersonName` behavior I've left alone.

Both were able to knock the expandable arrow out of the row, so set weights.

File: app/src/main/java/com/jerboa/ui/components/home/Home.kt [hunk 1]
```diff
@@ -436,13 +441,28 @@ fun AvatarAndAccountName(myPerson: Person?, showAvatar: Boolean) {
...
-            person = myPerson,
-            color = MaterialTheme.colorScheme.onSurface,
-        )
+        Column() {
+            PersonName(
+                person = myPerson,
+                color = MaterialTheme.colorScheme.onSurface,
+            )
+            Text(
+                text = myPerson?.let { federatedNameShown(it) } ?: run { "" },
```
Comment (MV-GH) [inline app/src/main/java/com/jerboa/ui/components/home/Home.kt:450]: just `?: ""`

Comment (camporter) [reply app/src/main/java/com/jerboa/ui/components/home/Home.kt]: Thanks, done :+1:

**Comment by MV-GH** [comment]
Greek string file should be updated now too

**Review by dessalines — APPROVED** [review]
Looks good, thanks!

**Pull request merged**